src/distribution/laplace.rs

```diff
@@ -0,0 +1,363 @@
+use distribution::{Continuous, Univariate};
+use rand::distributions::Distribution;
+use rand::distributions::OpenClosed01;
+use rand::Rng;
+use statistics::*;
+use std::f64;
+use {Result, StatsError};
+
+/// Continuous Univariate Laplace distribution.
+/// The Laplace distribution is a distribution over the real numbers parameterized by a mean and
+/// scale parameter. The PDF is:
+///     p(x) = \frac{1}{2 * scale} \exp{- |x - mean| / scale}.
+/// <a href="http://en.wikipedia.org/wiki/Laplace_distribution">Wikipedia - Laplace distribution</a>.
+pub struct Laplace {
+    location: f64,
+    scale: f64,
+}
+
+impl Laplace {
+    fn sample_unchecked<R: Rng + ?Sized>(r: &mut R, location: f64, scale: f64) -> f64 {
```

Comment:
I don't believe there's a need to split this function out from the `impl Distribution` trait as that `sample` method simply calls this one